src/PageController.php

```diff
@@ -356,8 +356,10 @@ public function is_connected_page() {
 
 		// Disable embed on the block editor.
 		$current_screen = did_action( 'current_screen' ) ? get_current_screen() : false;
-		if ( method_exists( $current_screen, 'is_block_editor' ) && $current_screen->is_block_editor() ) {
-			$is_connected_page = false;
+		if ( $current_screen !== false ) {
```

Comment:
Minor nitpick, but could we make this one line?
```
if ( $current_screen !== false && method_exists( $current_screen, 'is_block_editor' ) && $current_screen->is_block_editor() {}
```

Reply:
Thank you for looking at this pull request. I checked in the change you requested.